### ISSUE TITLE
Development: Homepage updates

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-mwcir#27]
Existing homepage elements reorganized for improved user experience.

Updates will be prioritized based on the total hours available.

**Comment by MirandaEcho** [comment]
- [ ] Update top layout - 2 hours
- [ ] Style widget boxes - 2 hours
- [ ] Style the actual widgets - 2 hours

Contact forms on hold pending budget.

**Comment by benlk** [comment]
![Screen Shot 2020-02-17 at 14 08 44](https://user-images.githubusercontent.com/1754187/74680569-0a1bf900-518f-11ea-946e-4dff49b23ff2.png)

Should the text not wrap beneath the image?

**Comment by kaylima** [comment]
@benlk ideally it would not wrap, but let's keep as-is for now and revisit once everything else is complete. it's a "nice to have" in terms of how we use the allotted time.
